site/archive-body.php

```diff
@@ -89,6 +87,19 @@
                             <?php echo wp_kses_post( $item->post_content ) ?>
                         <?php endforeach; ?>
                     </div>
+                    <?php
+                    $hex = p4r_get_theme_hex( $porch_fields['theme_color']['value'] );
+
+                    // Check for custom theme hex
+                    if ( isset( $porch_fields['custom_theme_color']['value'] ) && ! empty( $porch_fields['custom_theme_color']['value'] ) ){
+                        $hex = $porch_fields['custom_theme_color']['value'];
+                    }
+
+                    if ( $porch_fields['show_prayer_timer']['value'] === 'yes' ) : ?>
+                    <div class="mt-5">
+                        <?php show_prayer_timer( [ 'color' => $hex, 'duration' => 15] ); ?>
```

Comment:
@prykon Please add a check to make sure the function exists.
Will throw an error if the campaigns plugin isn't installed or up to date.